src/WebJobs.Script.WebHost/Middleware/NonAppServiceWorkerMiddleware.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.WebJobs.Script.Extensions;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Middleware
+{
+    public class NonAppServiceWorkerMiddleware
```

Comment:
This middleware modifies requests coming from App Service Front Ends and performs fixups. Recommend renaming this to AppServiceHeaderNormalizationMiddleware

Reply:
Yeah, I was worried that this may make it sound like it's for when running **on** not the otherway around, but I'm bad with naming. I'll update the name.